src/specs/VideoNativeComponent.ts

```diff
@@ -296,11 +296,18 @@ export type OnAudioFocusChangedData = Readonly<{
 }>;
 
 type ControlsStyles = Readonly<{
+  hidePosition?: WithDefault<boolean, false>;
+  hidePlayPause?: WithDefault<boolean, false>;
+  hideForward?: WithDefault<boolean, false>;
+  hideRewind?: WithDefault<boolean, false>;
+  hideNext?: WithDefault<boolean, false>;
```

Comment:
I agree with hiding the previous and next buttons by default. However, I don't understand why we would allow the user to hide the play and pause buttons.

Additionally, I think it would be better to implement this in full-screen mode, because if the user hides the forward button in normal mode but then sees it in full-screen mode, it could be confusing.

WDYT? @freeboub @KrzysztofMoch

Reply:
Background videos - it is just an option
> I wonder if the next and previous buttons should be hidden by default

I agree we can hide them by default

Reply:
@seyedmostafahasani depending of UI requierements, it make sense (live content, or not pausable video , why not :) )
@moskalakamil can you just change hideNext and  hidePrevious to true

Reply:
@freeboub 
Thank you for the explanation.
@moskalakamil 
Thank you for creating this PR.